task/move.go

```diff
@@ -109,21 +114,33 @@ func MaybeMove(fetch MaybeFetcher, put MaybePutter) error {
 // early then MoveCtx returns ctx.Err() just as MaybeMove returns any errors
 // that it encounters.
 func MoveCtx(ctx context.Context, fetch Fetcher, put Putter) error {
-	// [TODO]
+	ch := make(chan Thing)
+	ctx, cancelFn := context.WithCancel(ctx)
+	defer cancelFn()
```

Comment:
These are unnecessary. It's OK to just use the incoming context.

WithCancel copies everything from the ctx that you pass to it and then returns a new context with the ability to stop it earlier. Because you're only calling `cancel()` right at the end it has no effect at all. You would use WithCancel if you were then going to pass the context to a different function that you want to cancel. Remind me to chat with you about this because it can be confusing.

Reply:
okay